NEWS.rst

```diff
@@ -5,6 +5,119 @@ User-Visible Changes in Tahoe-LAFS
 ==================================
 
 .. towncrier start line
+Release 1.14.0 (2020-01-17)
+'''''''''''''''''''''''''''
+
+Removed Features
+----------------
+
+- "tahoe rm", an old alias for "tahoe unlink", has been removed. (`#1827 <https://github.com/tahoe-lafs/tahoe-lafs/issues/1827>`_)
```

Comment:
Hm.  All of the issue links are broken, pointing at GitHub issues instead of trac issues.